.circleci/config.yml

```diff
@@ -13,12 +13,12 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Merge with master
+          name: Merge with parent branch
           command: |
             git config user.name "dummy"
             git config user.email "dummy@example.org"
-            git fetch origin master
-            git merge --squash origin/master
+            git fetch origin $(circleci-branch) 
```

Comment:
I think it would make sense to avoid calling circleci-branch twice if possible:

```bash
parent_branch=$(circleci-branch)
git fetch origin $parent_branch
git merge --squash origin/$parent_branch
```

Reply:
8aa6d2d